db/contacts.json

```diff
@@ -0,0 +1,84 @@
+{
+"contacts":  [{
```

Comment:
This should be an object of object with keys for each contact (just like it will be when you get it back from firebase).

Reply:
Also, go ahead and make each contact unique.